session_0/item.rb

```diff
@@ -23,14 +23,20 @@ def initialize(name: '', price: 0, quantity: 1, category_id: 4,
   # Returns a boolean value whether than item is discounted i.e. the
   # discount deadline has been crossed or not.
   def discounted?
-    raise NotImplementedError # TODO
+    return !(Time.now > @discount_deadline)
```

Comment:
Now, to follow from the previous comment: Not (A > B) is same as A <= B.

`TIme.now <= discount_deadline` makes sense - if the current time is less than deadline, the item is discounted